### PR TITLE
First draft of onboarding documentation

### PR DESCRIPTION
**Pull request by pote**
This is a first stab at new user onboarding documentation

@foca @bradybd @Adonya @xxuejie any feedback is much appreciated.

### CODE REVIEW

**Comment by foca** [comment]
I think screenshots at least are a good idea. GIFs under ~5 seconds are also nice. Longer than that and it's annoying to have to wait until it gets to where you're stuck.

Longer video guides could be a good idea, but that's a whole other can o' worms.

**Comment by pote** [comment]
Good, I was thinking of extremely simple under 5 Second gifs at least for things that have complex interaction, such as defining a policy.

File: getting_started.md [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+
+## Overwatch Devices
+
+Overwatch uses an installable agent to keep tabs on your devices, analyzing and logging it's regular behavior and enforcing the security rules that you configure on our web application. Devices refer to any platform in which our client agent can be installed, as such they are the main entity of focus in our web interface.
+
+## The Overwatch Agent
+
+Our agent runs as a service in your device, it's purpose it's two-fold: it enforces any [Security Policy rules](#security-policies) that you set up, and it sends regular status updates to the Overwatch servers for analysis.
+
+The stats sent out by the agent include plain keepalives, inbound and outbound data traffic, open connections and general details such as its IP. The privacy of our user's data is as a main concern at Overwatch: communications between the agent and the overwach server are encrypted, and while we do analyze the behaviour of the device - such as the amounts of traffic exchanged between the device and the network - the actual data is never sent down the wire. If you have any concerns about privacy issues, please [get in touch](get-in-touch) and we'll be happy to discuss any doubts.
```
Comment (xxuejie) [inline getting_started.md]: Besides privacy concern regarding traffic, I think it is also worth mentioning that our agent has to run with root privilege, which might mean a lot to certain user.

Comment (xxuejie) [inline getting_started.md]: Besides privacy concern regarding traffic, I think it is also worth mentioning that our agent has to run with root privilege, which might mean a lot to certain user.

File: getting_started.md [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+```
+
+#### Starting the agent manually.
+
+The Overwatch agent is installed as a service in your device, as such, you'll be able to issue standard service commands to control it's behavior.
+
+```bash
+$ sudo service start overwatchd
+$ sudo service restart overwatchd
+$ sudo service stop overwatchd
```
Comment (xxuejie) [inline getting_started.md]: Forgive me if I'm not seeing the whole picture, but shouldn't the command be:

``` bash
$ sudo service overwatchd start
$ sudo service overwatchd restart
$ sudo service overwatchd stop
```

Or maybe we are talking about different things here?

File: getting_started.md [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+```
+
+#### Supported Devices
+
+Overwatch targets IoT platforms so this list will hopefully expand in the near future, our bootstrap script currently supports:
+
+* upstart-based Ubuntu Linux (by default, Ubuntu 14.10 and older).
```
Comment (xxuejie) [inline getting_started.md]: I thought we are now systemd-based?

**Comment by kristinpeterson** [comment]
@pote Great work! I made some minor copy updates.

File: getting_started.md (hunk 2 above)
Comment (pote) [inline getting_started.md]: No, you're correct! I wrote it from memory and switched the arguments. Will correct it now :)

File: getting_started.md (hunk 1 above)
Comment (pote) [inline getting_started.md]: You are correct, I'll update with something along those lines.

File: getting_started.md (hunk 3 above)
Comment (pote) [inline getting_started.md]: We currently support upstart, there's a PR waiting to be merged with systemd support (I'll update this document when merged)

**Comment by pote** [comment]
Thanks @kristinpeterson! Absolutely needed some editorial love.

**Comment by Adonya** [comment]
@pote I love the idea of short 5 second gifs. Also, I made a few edits and created a new PR, when you get a chance please review.

**Comment by kristinpeterson** [comment]
Big :+1: for 5 sec gifs!

**Comment by trotttrotttrott** [comment]
@pote looks great!

This is out of scope but I wonder if we should create an official (or unofficial) community Chef cookbook for Overwatch - which we kind of already have. It's nice to be able to read recipes when learning how to run new things. The Prometheus installation docs have a [Using Configuration Management Systems section](http://prometheus.io/docs/introduction/install/#using-configuration-management-systems) which is where I'm stealing this idea from.

**Comment by xxuejie** [comment]
@trotttrotttrott Huge :+1: on the idea.

File: getting_started.md [hunk 4]
```diff
@@ -0,0 +1,151 @@
...
+
+Overwatch uses an installable client agent to monitor your devices, analyzing and logging it's regular behavior and enforcing the security rules that you configure on our web application. The term `Device` refers to any platform or physical hardware on which our client agent can be installed.
+
+## The Overwatch Agent
+
+Our agent runs as a service on your device, it's purpose is two-fold: it enforces any [Security Policy rules](#security-policies) that you've defined in the web application, and it sends regular status updates to the Overwatch servers for analysis.
+
+The stats sent out by the agent include plain keepalives, inbound and outbound data traffic, open connections and general details such as the device's IP. The privacy of our user's data is very important to us, as such all communication between the agent and the Overwach server is encrypted, and while we do analyze the behaviour of the device - such as the amount of traffic exchanged between the device and the network, the actual data is never sent down the wire. If you have any concerns about privacy, please [get in touch](get-in-touch) and we'll be happy to discuss our security measures in more detail.
```
Comment (foca) [inline getting_started.md]: The `get in touch` link should be `[get in touch][get-in-touch]`. Using parens means that's the explicit URL. Callouts to links elsewhere are done with square brackets.

File: getting_started.md [hunk 5]
```diff
@@ -0,0 +1,151 @@
...
+
+#### Supported Devices
+
+Overwatch bootstrap script currently supports:
+
+* upstart-based Ubuntu Linux (by default, Ubuntu 14.10 and older)
+
+We have plans to support additional IoT focused platofrms and hardware and in the near future. In the meantime, if you'd like to run Overwatch in a platform that is not listed here [let us know](get-in-touch) and we'll look into adding it to our bootstrap process.
+
+## The Overwatch Web Interface
```
Comment (foca) [inline getting_started.md:91]: This whole section should probably be moved to another file. Simpler doc pages that deal with a single thing are better than having to scan through a very long file to find something.

**Comment by foca** [comment]
@trotttrotttrott I like the idea of the cookbooks :)

@pote I wonder if it'd make sense to re-take the screencaps now that we have threshold-based policy rules merged in.

**Comment by pote** [comment]
@trotttrotttrott the idea about cookbooks is nothing short of f**\* genius, we could also include Ansible recipes (since we know that) and maybe puppet in the future, they are fantastic ways to help out possible users get up and running.

File: getting_started.md (hunk 4 above)
Comment (pote) [inline getting_started.md]: Correct! fixing

**Pull request merged**